.github/workflows/publish.yml

```diff
@@ -0,0 +1,31 @@
+name: publish
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - uses: ruby/setup-ruby@477b21f02be01bcb8030d50f37cfec92bfa615b6
+        with:
+          ruby-version: 2.6
+      - name: Install dependencies
+        run: |
+          pip install bikeshed xml2rfc python-dateutil
```

Comment:
I think we can leave Bikeshed out of the publish.yml flow, and instead copy https://github.com/martinthomson/i-d-template/blob/master/template/.github/workflows/publish.yml over literally.

Reply:
Thanks. Fixed.
I literally copied the template.